tests/helpers.test.ts

```diff
@@ -118,10 +129,87 @@ describe('ModuleHelpers', () => {
 
   describe('isCRATestCommand', () => {
     it('should return true for CRA', () => {
-      expect(isCRATestCommand('react-scripts test --env=jsdom')).toBeTruthy()
+      expect(isCRATestCommand('react-scripts test --env=jsdom')).toBe(true)
     })
+
     it('should return false for other scripts', () => {
-      expect(isCRATestCommand('custom-script')).toBeFalsy()
+      expect(isCRATestCommand('custom-script')).toBe(false)
+    })
+  })
+
+  describe('pathToJest', () => {
+    const defaultSettings: any = {
+      pathToJest: null,
+      rootPath: '',
+    }
+
+    it('returns "npm test --" when bootstrapped with create-react-app', () => {
+      mockReadFileSync.mockReturnValueOnce(
+        JSON.stringify({
+          scripts: {
+            test: 'react-scripts test',
+          },
+        })
+      )
+
+      expect(pathToJest(defaultSettings)).toBe('npm test --')
+    })
+
+    it('returns the normalized "pathToJest" setting when set by the user', () => {
+      const expected = {}
+      mockNormalize.mockReturnValueOnce(expected)
+
+      const settings: any = {
+        pathToJest: 'jest',
```

Comment:
It's just a small thing, but maybe we should test for a different value since `jest` is the default outcome of the `pathToJest` function.

Reply:
Good call. I've switched it to a `{}`.